docs/user-guide/node-api.md

```diff
@@ -96,118 +94,101 @@ It resolves with an object (see [The returned promise](#the-returned-promise)) t
 As `config` contains no relative paths for `extends` or `plugins`, you do not have to use `configBasedir`:
 
 ```js
-stylelint
-  .lint({
+try {
+  const result = await stylelint.lint({
```

Comment:
minor
suggestion

IIRC top-level await is not supported by node in CJS modules.
Since this version will support both, should we have a compatible example to permit easy copy paste?
i.e. use `then`/`catch` instead (like the previous example)

Reply:
Yes, top-level await is unsupported in CJS, but I don't think we have to stick to whether top-level await or not because these examples are just code snippets.

Personally, I don't prefer `then`/`catch` so much since there are many nestings.

Reply:
I guess, it can be considered a subtle nudge to move towards ESM before v17.